docs/migrations/web/MIGRATION-v3.md

```diff
@@ -0,0 +1,26 @@
+# Migration Guide
+
+Introducing version 3 of the _spirit-web_ package.
+
+> Please follow these steps to safely upgrade to Spirit Design System v3 components.
+
+## Overview
+
+- [Component Changes](#component-changes)
+  - [Button and ButtonLink: renamed modifier `square`](#button-and-buttonlink-square-modifier-renamed-to-symmetrical)
+
+## Component Changes
+
+### Button and ButtonLink: `square` modifier renamed to `symmetrical`
+
+Button `isSquare` prop was renamed to `isSymmetrical`.
+
+#### Migration Guide
+
+Instead of using `.Button--square` or `.ButtonLink--square`, use a `.Button--symmetrical` or
+`.ButtonLink--symmetrical`.
```

Comment:
I don't think we have a ButtonLink in web package. Please remove any reference from this file.